src/Features/CSharp/Portable/MakeTypeAbstract/CSharpMakeTypeAbstractCodeFixProvider.cs

```diff
@@ -8,41 +8,38 @@
 using System.Diagnostics.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.MakeClassAbstract;
+using Microsoft.CodeAnalysis.MakeTypeAbstract;
 
-namespace Microsoft.CodeAnalysis.CSharp.MakeClassAbstract
+namespace Microsoft.CodeAnalysis.CSharp.MakeTypeAbstract
 {
-    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(CSharpMakeClassAbstractCodeFixProvider)), Shared]
-    internal sealed class CSharpMakeClassAbstractCodeFixProvider : AbstractMakeClassAbstractCodeFixProvider<ClassDeclarationSyntax>
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(CSharpMakeTypeAbstractCodeFixProvider)), Shared]
+    internal sealed class CSharpMakeTypeAbstractCodeFixProvider : AbstractMakeTypeAbstractCodeFixProvider<TypeDeclarationSyntax>
     {
         [ImportingConstructor]
         [SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
-        public CSharpMakeClassAbstractCodeFixProvider()
+        public CSharpMakeTypeAbstractCodeFixProvider()
         {
         }
 
         public override ImmutableArray<string> FixableDiagnosticIds { get; } =
                ImmutableArray.Create(
-                   "CS0513" // 'C.M()' is abstract but it is contained in non-abstract class 'C'
+                   "CS0513" // 'C.M()' is abstract but it is contained in non-abstract type 'C'
```

Comment:
I am assuming the analyzer was updated for this and the message updated, isn't it?

Reply:
@Evangelink Yes the message for CS0513 was updated.